pages/view/_iscnId.vue

```diff
@@ -332,14 +340,21 @@
                 :label="$t('iscn.meta.creator.name')"
                 class="w-[50%] my-[12px]"
               >
-                <Label :text="stakeholderInfo.authorName" tag="div" preset="p4" />
+                <Link
+                  v-if="stakeholderInfo.authorWalletAddresses.length"
```

Comment:
Why check for `stakeholderInfo.authorWalletAddresses.length` when the Link is about `stakeholderInfo.authorName`?

Reply:
I think it's legacy code between version iteration